inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/WorkflowApproverServiceImpl.java

```diff
@@ -89,11 +92,18 @@ public Integer save(ApproverRequest request, String operator) {
     @Override
     public ApproverResponse get(Integer id) {
         Preconditions.expectNotNull(id, "approver id cannot be null");
+
         WorkflowApproverEntity approverEntity = approverMapper.selectById(id);
         if (approverEntity == null) {
             LOGGER.error("workflow approver not found by id={}", id);
             throw new BusinessException(ErrorCodeEnum.WORKFLOW_APPROVER_NOT_FOUND);
         }
+
+        UserInfo userInfo = LoginUserUtils.getLoginUser();
+        boolean isAdmin = userInfo.getRoles().contains(UserTypeEnum.ADMIN.name());
+        Preconditions.expectTrue(isAdmin || approverEntity.getApprovers().contains(userInfo.getName()),
```

Comment:
Suggest use `userService.checkUser(approverEntity.getApprovers(), userInfo.getName(), "Current user does not have permission to get this workflow approver info")`

Reply:
fix it